pod-batch.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+# this value should be differernt when running script again, as each time a script is run a new Pod is created
+  name: pod1
```

Comment:
Can not expect the script to be manually updated.  What is the alternative?  Is there an environment variable to use?